docs/examples.rst

```diff
@@ -333,7 +333,7 @@ The following example demonstrates a few things:
 
     store_var_contract = w3.eth.contract(address=address, abi=contract_interface["abi"])
 
-    gas_estimate = store_var_contract.functions.setVar(255).estimateGas()
+    gas_estimate = store_var_contract.functions.setVar(255).estimate_gas()
```

Comment:
This should still be `estimateGas`. It's a little confusing because we have an `estimateGas` method on the contract instance, and in the eth module. In this PR, the contract methods haven't changed, just `w3.eth.estimateGas` from the eth module. Let me know if that doesn't make sense!